tests/phases/design/test_electrical_design.py

```diff
@@ -324,3 +327,29 @@ def test_cost_crossing():
     cross_sim.run()
 
     assert cross_sim.crossing_cost != base_sim.crossing_cost
+
+
+def test_deprecated_landfall():
+
+    base = deepcopy(config)
+    deprecated = deepcopy(config)
+
+    deprecated["landfall"] = {"interconnection_distance": 4}
+
+    with catch_warnings(record=True) as w:
+
+        sim = ElectricalDesign(base)
+        sim.run()
+
+        assert len(w) == 0
+
+        sim = ElectricalDesign(deprecated)
+        sim.run()
+
+        assert len(w) == 1
+        assert issubclass(w[0].category, DeprecationWarning)
+        assert (
+            str(w[0].message)
+            == "landfall dictionary will be deprecated and moved \
+                    into [export_system_design][landfall]."
+        )
```

Comment:
This could be simplified to be more like:

```python
with pytest.warns(DeprecationWarning):
    sim = ElectricalDesign(deprecated)
    sim.run()
```

Reply:
Oh. Neat. I can clean those up.